lib/rubocop/cop/style/multiline_when_then.rb

```diff
@@ -35,7 +35,7 @@ def on_when(node)
           return if !node.children.last.nil? && !node.multiline? && node.then?
 
           # With more than one statements after then, there's not offense
-          return if node.children.last&.begin_type?
+          return if accept_node_type?(node.children.last)
```

Comment:
How about `node.body` instead of `node.children.last`?

Reply:
That is more readable. Thanks!